GameOfThrones/Controllers/HouseController.cs

```diff
@@ -0,0 +1,39 @@
+﻿using GameOfThrones.Models;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Web;
+using System.Web.Mvc;
+
+namespace GameOfThrones.Controllers
+{
+    public class HouseController : Controller
+    {
+        private CastleContext db = new CastleContext();
+
+        // GET: House
+        public ActionResult Index()
+        {
+            return View(db.Houses.ToList());
+        }
+
+        public ActionResult Characters(int? id)
+        {
+            if (id == null)
+            {
+                return new HttpStatusCodeResult(System.Net.HttpStatusCode.BadRequest);
+            }
+
+            var characters = from c in db.Characters
+                             where c.HouseId.HouseId == id
+                             select c;
```

Comment:
Why use this instead of
```
var characters = db.Characters.Select(c => c.HouseId.HouseId == id).ToList()
```
Although, fair play for using this form of querying though